omnibus/resources/agent/msi/source.wxs.erb

```diff
@@ -720,8 +644,39 @@
     <WixVariable Id="WixUIDialogBmp" Value="Resources\assets\dialog_background.bmp" />
     <WixVariable Id="WixUIBannerBmp" Value="Resources\assets\banner_background.bmp" />
 
-    <CustomAction Id='SetFinalizeProperty' Return='check' Property='FinalizeInstall' 
-        Value='PROJECTLOCATION=[PROJECTLOCATION];APPLICATIONDATADIRECTORY=[APPLICATIONDATADIRECTORY];DDAGENTUSER_NAME=[DDAGENTUSER_NAME];DDAGENTUSER_PASSWORD=[DDAGENTUSER_PASSWORD];SYSPROBE_PRESENT=[SYSPROBE_PRESENT];ADDLOCAL=[ADDLOCAL]' />
+    <!-- Properties must each be on their own line and end with a semicolon ;
+      NOTE: This property will cause the build time warning
+      "warning LGHT1076 : ICE03: String overflow (greater than length permitted in column);"
+      This does not affect the final artefact, as it seems to affects only the base string, not the expanded one.
+      At runtime this limitation doesn\'t seem to apply.
+    -->
+    <CustomAction Id='SetFinalizeProperty' Return='check' Property='FinalizeInstall'
+        Value='
+            PROJECTLOCATION=[PROJECTLOCATION];
+            APPLICATIONDATADIRECTORY=[APPLICATIONDATADIRECTORY];
+            DDAGENTUSER_NAME=[DDAGENTUSER_NAME];
+            DDAGENTUSER_PASSWORD=[DDAGENTUSER_PASSWORD];
+            SYSPROBE_PRESENT=[SYSPROBE_PRESENT];
+            ADDLOCAL=[ADDLOCAL];
+            APIKEY=[APIKEY];
+            TAGS=[TAGS];
+            HOSTNAME=[HOSTNAME];
+            PROXY_HOST=[PROXY_HOST];
+            PROXY_PORT=[PROXY_PORT];
+            PROXY_USER=[PROXY_USER];
+            PROXY_PASSWORD=[PROXY_PASSWORD];
+            LOGS_ENABLED=[LOGS_ENABLED];
+            APM_ENABLED=[APM_ENABLED];
+            PROCESS_ENABLED=[PROCESS_ENABLED];
+            CMD_PORT=[CMD_PORT];
+            SITE=[SITE];
+            DD_URL=[DD_URL];
+            LOGS_DD_URL=[LOGS_DD_URL];
+            PROCESS_DD_URL=[PROCESS_DD_URL];
+            TRACE_DD_URL=[TRACE_DD_URL];
+            PYVER=[PYVER];
+            HOSTNAME_FQDN_ENABLED=[HOSTNAME_FQDN_ENABLED];
```

Comment:
It would be nice to have these sorted either in importance or alphabetically so that maintenance of these vars is a bit easier.